shared/artifact-links.ts

```diff
@@ -27,6 +27,8 @@ export const artifactLinks = {
 
     'KeymanDesktop_TestPullRequests': {platform: 'windows', name: 'Windows', icon: 'windows.png', downloads: [
       {fragment: 'release/$version/keyman-$version.exe', name: '**Keyman for Windows**', icon: 'keyman.png'} ,
+      {fragment: 'support/editor32.exe', name: 'Text Editor (32 bit)', icon: 'tool.png'} ,
+      {fragment: 'support/editor64.exe', name: 'Text Editor (64 bit)', icon: 'tool.png'} ,
```

Comment:
Maybe I'm looking in the wrong place. I didn't see support/editor*.exe artifacts in

this [PR build](https://build.palaso.org/buildConfiguration/KeymanDesktop_TestPullRequests/382333?buildTab=artifacts#%2Fsupport;%2Frelease)

Reply:
See https://github.com/keymanapp/keyman/pull/8750 as the first one with the editor artifacts